edx_proctoring/static/proctoring/js/proctored_app.js

```diff
@@ -5,4 +5,16 @@ $(function() {
         model: new ProctoredExamModel()
     });
     proctored_exam_view.render();
+
+    // show the timer bar when the window scrolls.
+    $(window).scroll(function() {
+        if ($(this).scrollTop() > 60){
+            $(".proctored_exam_status").addClass('fixed');
+            $(".wrapper-course-material").css('margin-top', '58px');
```

Comment:
There is a disrupt change in the course navigation when the window scroll position matches the timer bar position.